### ISSUE TITLE
dart-sass-embedded ENOENT on docker

### ISSUE THREAD

**Issue by krystofbe** [sass/embedded-host-node#143]
hi! Thanks for this awesome library. Everything works fine on my Mac, but when I try to use `sass-embedded` in my `Dockerfile` I get the error

```
$ webpack
#11 2.215 node:events:505
#11 2.215       throw er; // Unhandled 'error' event
#11 2.215       ^
#11 2.215 
#11 2.215 Error: spawn /app/node_modules/sass-embedded/dist/lib/src/vendor/dart-sass-embedded/dart-sass-embedded ENOENT
#11 2.215     at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
#11 2.215     at onErrorNT (node:internal/child_process:478:16)
#11 2.215     at processTicksAndRejections (node:internal/process/task_queues:83:21)
#11 2.215 Emitted 'error' event on ChildProcess instance at:
#11 2.215     at Process.ChildProcess._handle.onexit (node:internal/child_process:289:12)
#11 2.215     at onErrorNT (node:internal/child_process:478:16)
#11 2.215     at processTicksAndRejections (node:internal/process/task_queues:83:21) {
#11 2.215   errno: -2,
#11 2.215   code: 'ENOENT',
#11 2.215   syscall: 'spawn /app/node_modules/sass-embedded/dist/lib/src/vendor/dart-sass-embedded/dart-sass-embedded',
#11 2.215   path: '/app/node_modules/sass-embedded/dist/lib/src/vendor/dart-sass-embedded/dart-sass-embedded',
#11 2.215   spawnargs: []
#11 2.215 }
#11 2.246 error Command failed with exit code 1.
#11 2.246 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
------
```

this is strange, because the file exists on the image:

```
/app # ls -alh /app/node_modules/sass-embedded/dist/lib/src/vendor/dart-sass-embedded/dart-sass-embedded
-rwxr-xr-x    1 root     root        8.6M Apr 26 02:41 /app/node_modules/sass-embedded/dist/lib/src/vendor/dart-sass-embedded/dart-sass-embedded
```

I've made a sample repo to reproduce the error: https://github.com/krystofbe/embedded-sass-test

**Comment by ntkme** [comment]
The ENOENT is thrown by the `ld-linux`, the reason is that the base image `alpine` is using `musl-libc`, for which dart is not compatible with. If you want to see the actual error try `ldd /app/node_modules/sass-embedded/dist/lib/src/vendor/dart-sass-embedded/dart-sass-embedded`.

The solution would be use a normal node image that is not `alpine` based.

**Comment by ntkme** [comment]
We should probably add a check for `musl-libc` during installation using the method from node-sass and explicitly throw an error saying `musl-libc` is not supported: https://github.com/sass/node-sass/blob/15233309452f1a33896fe08fcbd3cdc0294c7818/lib/extensions.js#L419-L441

**Comment by nex3** [comment]
Fixed by sass/embedded-host-node#149
